engine/events/magnet_train.asm

```diff
@@ -4,13 +4,13 @@ MagnetTrain: ; 8cc04
 	jr nz, .ToGoldenrod
 	ld a, 1 ; forwards
 	lb bc,  $40,  $60
-	lb de, (11 * 8) - (11 * 8 + 4), -$60
+	lb de, ((11 * 8) - (11 * 8 + 4)), -$60
 	jr .continue
 
 .ToGoldenrod:
 	ld a, -1 ; backwards
 	lb bc, -$40, -$60
-	lb de, (11 * 8) + (11 * 8 + 4), $60
+	lb de, ((11 * 8) + (11 * 8 + 4)), $60
```

Comment:
this is a problem with the macro, not the usage

Reply:
Should I bother undoing it, then?